.mailmap

```diff
@@ -14,163 +18,203 @@
 # Note that each qiskit element uses its own mailmap so it may be necessary to
 # propagate changes in other repos for consistency.
 #
-Abdón Rodríguez Davila <a@abdonrd.com> <a@abdonrd.com>
+Abby Mitchell <23662430+javabster@users.noreply.github.com> <abby.mitchell@btinternet.com>
+Abby Mitchell <23662430+javabster@users.noreply.github.com> <abby.s.mitchell@gmail.com>
+Abdón Rodríguez Davila <a@abdonrd.com>
 Abigail J. Cross <across@us.ibm.com>
+Abraham Asfaw <abraham.asfaw@ibm.com> <asfaw@princeton.edu>
+Adenilton Silva <7927558+adjs@users.noreply.github.com> <adjs@users.noreply.github.com>
+Adish Vartak <avartak@cern.ch> <adishvartak@gmail.com>
 Adrien Suau <adrien.suau@cerfacs.fr>
 Albert Frisch <albert.frisch@de.ibm.com>
 Albert Frisch <albert.frisch@de.ibm.com> <alfr@de.ibm.com>
-Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@users.noreply.github.com>
+Alejandro Montanez <alejomonbar@gmail.com>
+Alex Zhang <alex.lx.zhang@gmail.com>
+Ali Javadi-Abhari <ali.javadi@ibm.com>
+Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@princeton.com>
 Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@princeton.edu>
-Alejandro Pozas-Kerstjens <physics@alexpozas.com> <apozas@users.noreply.github.com>
+Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@users.noreply.github.com>
+Almudena Carrera Vazquez <acv@zurich.ibm.com>
 Almudena Carrera Vazquez <acv@zurich.ibm.com> <almudenacarreravazquez@hotmail.com>
 Andrew W. Cross <awcross@us.ibm.com>
-Andrew W. Cross <awcross@us.ibm.com> Andrew Cross <awcross@Andrews-MacBook-Pro.local>
-Andrew W. Cross <awcross@us.ibm.com> Andrew Cross <awcross@andrews-mbp.watson.ibm.com>
+Andrew W. Cross <awcross@us.ibm.com> <awcross@Andrews-MacBook-Pro.local>
+Andrew W. Cross <awcross@us.ibm.com> <awcross@andrews-mbp.watson.ibm.com>
+André Großardt <99898527+grossardt@users.noreply.github.com>
+André Großardt <99898527+grossardt@users.noreply.github.com> <andgros@einstein.tpi.uni-jena.de>
+Animesh N. Dangwal <animesh.leo@gmail.com>
 Anna Phan <9410731+attp@users.noreply.github.com> <anna.phan@au1.ibm.com>
 Annanay Kapila <k.annanay@gmail.com>
+Anthony Gandon <anthony.gandon@ibm.com>
+Anthony Gandon <anthony.gandon@ibm.com> <antho91370@gmail.com>
+Anthony Gandon <anthony.gandon@ibm.com> <nan@zurich.ibm.com>
+Anton Dekusar <62334182+adekusar-drl@users.noreply.github.com>
+Anton Dekusar <62334182+adekusar-drl@users.noreply.github.com> <adekusar@ie.ibm.com>
```

Comment:
The canonical email here might want flipping to be the IBM one.

Reply:
Same as above.